test/unit_Ellipsoid.jl

```diff
@@ -55,6 +55,6 @@ for N in [Float64, Float32]
     @test σ(d, E) ≈ N[1., 2. - sqrt(2)]
 
     # an_element and set membership functions
-    E = Ellipsoid(N[1., 2.], 2*eye(N, 2))
+    E = Ellipsoid(N[1., 2.], Matrix{N}(N(2.) * I, 2, 2))
```

Comment:
consider the slightly shorter `Matrix{N}(2I, n, n)`

Reply:
Very good:
```julia
julia> @time Matrix{N}(N(2.) * I, 2, 2)
  0.000095 seconds (7 allocations: 304 bytes)

julia> @time Matrix{N}(2I, 2, 2)
  0.000090 seconds (6 allocations: 288 bytes)
```